packages/cadl-vscode/package.json

```diff
@@ -21,7 +21,7 @@
   "categories": [
     "Programming Languages"
   ],
-  "type": "commonjs",
+  "type": "module",
```

Comment:
oh, this works now for the vscode extension! Hooray. I think there are some workarounds that could be removed.

Reply:
not really, this just says that we wrote the code with modules but it still gets compiled to commonjs in rollup.js

Reply:
let me also double check everything works fine with the extension here

Reply:
OK, check F5 too where we run extension without rollup

Reply:
though maybe it also just work with module, let me try that too

Reply:
yeah unfortunately it's still not compatible with modules